docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -213,7 +213,7 @@ You can change the density of the rows and the column header.
 
 ### Size prop
 
-There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the `DataGrid` is set to `medium`. The default grid density can be changed by applying the `size` prop. The `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
+There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the data grid is set to `medium`. The default grid density can be changed by applying the `size` prop. The `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
```

Comment:
That's odd, I left a comment with a suggestion that fixed some of the issues with this description, while leaving some of it open for discussion. Could we please not just overrule in those cases?

Reply:
Thanks for adding the suggestion back @oliviertassinari. The comment was something to the effect that it isn't necessary to reproduce the API documentation in the description. This change also resolves the potential confusion around what the options actually apply to, and also the use of "default" to mean two different things.

Reply:
I'm sorry about that @mbrookes I did merge it. I think git merges it weirdly if there are 2 suggestions on the same block.

Reply:
Good to know, I'll have to keep an eye out for that, thanks.